docs/workflow_tasks/move_granules.md

```diff
@@ -0,0 +1,60 @@
+---
+id: move_granules
+title: Move Granules
+hide_title: true
+---
```

Comment:
Just a note, this documentation may have to change once @mennovandiermen 's most recent PR goes through #883

Reply:
Right you are.    I'll coordinate with @laurenfrederick  and @mennovandiermen